openedx/core/djangoapps/user_api/accounts/api.py

```diff
@@ -517,3 +520,17 @@ def _validate_email(email):
         raise AccountEmailInvalid(
             u"Email '{email}' format is not valid".format(email=email)
         )
+
+
+def set_has_profile_image(username, has_profile_image=True):
```

Comment:
so the reason not to move has_profile_image into the uploading package is that it would create a direct dependency on the database / models where we do not have one presently.  (implies the shared database pattern)

I'm stuck on "allows anyone" - I don't think that is the case.  The function is not being exposed to network clients, only in-process.  I think that makes it like a lot of other functions in this module that provide read/write access to a particular attribute of a user account, but doesn't have any HTTP view in front of it, nor take an HTTP request as a parameter.  Let me know if I'm missing something?